src/index.md

```diff
@@ -12,6 +12,118 @@ The goal of this guide is to answer questions that developers of the JDK might h
 
 There are many common use cases that aren't detailed in the formal process. This guide suggests how to work in such cases.
 
+## I have a patch, what do I do?
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [Oracle Contributor Agreement (OCA)](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html)
+* [JDK Bug System (JBS)](https://bugs.openjdk.java.net/)
+* [OpenJDK Project Roles](https://openjdk.java.net/bylaws#project-roles)
+:::
+
+In many GitHub projects the standard way to propose a change is to create a pull request (PR) and discuss the patch in the PR. For OpenJDK projects the situation is somewhat different. First of all, the JDK and the surrounding tooling developed in the OpenJDK projects are part of commercial products maintained and sold by many companies. The JDK is installed on billions of devices and there are many millions of developers out there who use Java. For this reason there must be several safety checks before a change is made.
+
+### 1. Sign the OCA
+
+Oracle is the steward of the OpenJDK project, and owns the Java brand. In order to make your patch available for review you must first sign the [Oracle Contributor Agreement](https://www.oracle.com/technical-resources/oracle-contributor-agreement.html) (OCA). This agreement gives Oracle and you as a contributor joint copyright interests in the code. You will retain your copyright while also granting those rights to Oracle.
+
+When you sign the OCA, please make sure that you specify your GitHub user name in the `Username` field of the OCA. If you try to create a PR before you have signed the OCA, or if you didn't specify your GitHub user name, you'll get instructions telling you to do so, and the PR won't be published until this is done. OCA registration is a manual process. Please allow for up to several days to have your OCA application processed, even though it's normally processed swiftly.
+
+### 2. Socialize your change
+
+Once the OCA is signed, please restrain your urge to create a PR just a little while longer. In order to prepare the community for your patch, please socialize your idea on the relevant [mailing lists](#mailing-lists). To understand the criteria by which your patch is going to be judged, please read [_Why is My Change Rejected?_](#why-is-my-change-rejected) below. In short, hidden constraints and assumptions, stability and quality, maintainability, compatibility, and conformance to specifications must be considered before your PR is ready to be submitted. If you don't understand the constraints for acceptance, you might be surprised when your PR is rejected.
```

Comment:
One of the benefits of making the PR is that the openjdk bot tags the PR with the appropriate mailing list. Being a first time contributor, finding the relevant mailing list was not easy (there are just a lot of lists! 😅 ) - I was very relieved when the PR bot added the tag. I'm not suggesting this text be changed, just adding my experience.

Reply:
Maybe a better suggestion is to open the PR in draft mode? This means tthere is no requirement for a JBS issue number, and no RFR mail will be sent out -- but the bot still populates the proper tags, if I recall correctly.

I agree that getting assistance from the bot in finding a reasonable starting set of lists is good support for newcomers (even if it might need adjustment; but someone on the targeted lists can certainly help with that).

Reply:
There's this section in the Guide that intends to help finding the right list: https://openjdk.java.net/guide/#code-owners
@MariusVolkhart Is there information missing there that would have made it easier for you to find the right list?

Reply:
@JesperIRL No, it's all there. Thanks!